LaravelKenya/Laravel-Kenya
# Sets up admin panel using filament v3 stable

## FEATURE DESCRIPTION
sets up default admin panel using Laravel filament v3. With dark mode disabled.

### HOW TO TEST
1. Checkout to this branch (feat-setup-admin-panel)
2. composer install.
3. php artisan migrate
4. Create a filament user with the command php artisan make:filament-user 
5. Enter name, email and password as prompted.
6. php artisan serve.
7.  Navigate to localhost(http://127.0.0.1:800)/admin.
8. This form appears.
![image](https://github.com/LaravelKenya/Laravel-Kenya/assets/105051105/1fe3f2f7-6d54-494e-9437-86f4fdc3aa09)
9. use the credentials you created in step 4 and 5 to login to admin dashboard.

![image](https://github.com/LaravelKenya/Laravel-Kenya/assets/105051105/7fa204ea-fa35-401c-879c-dfb4a138802a)

### Protecting admin area in production.
The admin email will need to be whitelisted in the env for them to access the dashboard in production. Here a sample env config.
ADMIN_EMAIL=example@admin.com
